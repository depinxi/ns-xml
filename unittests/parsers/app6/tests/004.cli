-c "An exclusive group will NOT" -d "accept the presence of more than one sub option" 