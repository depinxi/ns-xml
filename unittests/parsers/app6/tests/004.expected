CLI: "-c", "An exclusive group will NOT", "-d", "accept the presence of more than one sub option"
Value count: 0
Values: 
Error count: 1
Subcommand: 
groupJ=
switchA=False
argumentB=
groupK=multiargC
multiargC=An exclusive group will NOT
argumentD=
groupL=
groupM=
switchE=False
argumentF=
groupN=
argumentG=
multiArgumentH=
